testsuite/.rubocop_todo.yml

```diff
@@ -588,14 +571,15 @@ Style/StringHashKeys:
     - 'features/support/constants.rb'
     - 'features/support/twopence_init.rb'
 
-# Offense count: 207
+# Offense count: 210
 # Cop supports --auto-correct.
 # Configuration parameters: EnforcedStyle, ConsistentQuotesInMultiline.
 # SupportedStyles: single_quotes, double_quotes
 Style/StringLiterals:
   Exclude:
     - 'Gemfile'
     - 'ext-tools/maintenance_crawler.rb'
+    - 'features/step_definitions/cobbler_steps.rb'
```

Comment:
Let's try to clean it up to don't include this, and in the future we will slowly remove the rest of here too. That one about using single quotes I think it's quite important for Ruby style.